pkg/apis/core/validation/validation_test.go

```diff
@@ -10285,14 +10353,14 @@ func TestValidateNode(t *testing.T) {
 		for i := range errs {
 			field := errs[i].Field
 			expectedFields := map[string]bool{
-				"metadata.name":         true,
-				"metadata.labels":       true,
-				"metadata.annotations":  true,
-				"metadata.namespace":    true,
-				"spec.externalID":       true,
-				"spec.taints[0].key":    true,
-				"spec.taints[0].value":  true,
-				"spec.taints[0].effect": true,
+				"metadata.name":                                                                                               true,
```

Comment:
gofmt changes should not be part of this PR. Do you use go 1.11?

Reply:
Yes, I am using  the same go version